Sources/Testing/ABI/v0/Encoded/ABIv0.EncodedIssue.swift

```diff
@@ -16,6 +16,19 @@ extension ABIv0 {
   /// assists in converting values to JSON; clients that consume this JSON are
   /// expected to write their own decoders.
   struct EncodedIssue: Sendable {
+    /// An enumeration representing the level of severity of a recorded issue.
+    ///
+    /// For descriptions of individual cases, see ``Issue/Severity-swift.enum``.
+    enum Severity: String {
```

Comment:
FWIW we could just use the `Issue.Severity` type here; we only need a separate type if it will encode in a different way (which I don't think is the case here?)

In any event, if you're keeping this enum, please mark it `Sendable`.

Reply:
I'm a bit wary of sharing types like that since I fear it might cause us to overlook something when evaluating whether changes have any ABI impact. I can certainly mark it `Sendable` though.